cgmes/cgmes-completion/src/main/java/com/powsybl/cgmes/completion/CreateMissingContainersPreProcessor.java

```diff
@@ -0,0 +1,216 @@
+/**
+ * Copyright (c) 2023, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+package com.powsybl.cgmes.completion;
+
+import com.google.auto.service.AutoService;
+import com.powsybl.cgmes.conversion.CgmesImportPreProcessor;
+import com.powsybl.cgmes.conversion.export.CgmesExportContext;
+import com.powsybl.cgmes.conversion.export.CgmesExportUtil;
+import com.powsybl.cgmes.conversion.export.elements.*;
+import com.powsybl.cgmes.model.CgmesModel;
+import com.powsybl.cgmes.model.CgmesNames;
+import com.powsybl.commons.PowsyblException;
+import com.powsybl.commons.config.PlatformConfig;
+import com.powsybl.commons.datasource.ZipFileDataSource;
+import com.powsybl.commons.io.WorkingDirectory;
+import com.powsybl.commons.parameters.Parameter;
+import com.powsybl.commons.parameters.ParameterDefaultValueConfig;
+import com.powsybl.commons.parameters.ParameterType;
+import com.powsybl.commons.reporter.Reporter;
+import com.powsybl.commons.xml.XmlUtil;
+import com.powsybl.iidm.network.Network;
+import com.powsybl.iidm.network.NetworkFactory;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.xml.stream.XMLStreamException;
+import javax.xml.stream.XMLStreamWriter;
+import java.io.IOException;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.Objects;
+import java.util.Set;
+import java.util.stream.Collectors;
+import java.util.zip.ZipEntry;
+import java.util.zip.ZipOutputStream;
+
+/**
+ * @author Luma Zamarreño <zamarrenolm at aia.es>
+ */
+@AutoService(CgmesImportPreProcessor.class)
+public class CreateMissingContainersPreProcessor implements CgmesImportPreProcessor {
+
+    public static final String NAME = "CreateMissingContainers";
```

Comment:
For homogenisation purposes, you can simply name it `missingContainers`

Reply:
But for example the IIDM modification post-processor that replaces tie lines by lines starts also with a verb (`replaceTieLinesByLines`). I prefer if we keep the action in the name. What do you think?

Reply:
I think there is a difference between modifications (which state the action they do) and processors (which state the context, particularly for CGMES processors) but both are okay in my opinion but it should start with a low case

Reply:
The name now starts with lower case, `createMissingContainers`.
I think of pre/post-processors as specialized tools carrying out specific tasks before/after CGMES import, so for me it is ok if their name starts with a verb.